nifi-nar-bundles/nifi-graph-bundle/nifi-other-graph-services/src/main/java/org/apache/nifi/graph/TinkerpopClientService.java

```diff
@@ -167,6 +167,7 @@ public class TinkerpopClientService extends AbstractControllerService implements
                     " Note: when using a remote.yaml file, this username value (if set) will overload any " +
                     "username set in the YAML file.")
             .required(false)
+            .addValidator(Validator.VALID)
```

Comment:
Seems like `StandardValidators.ATTRIBUTE_EXPRESSION_LANGUAGE_VALIDATOR` or `StandardValidators.NON_EMPTY_VALIDATOR` would be better for these properties - this allows them to remain optional, but prevents empty values being set, which is probably not valid